poc_mobile_verifier/src/subnetwork_rewards.rs

```diff
@@ -43,11 +42,11 @@ impl SubnetworkRewards {
             .map(|(pubkey, mut shares)| {
                 let speedmultiplier = speedtests
                     .get_average(&pubkey)
-                    .map_or(dec!(0.0), |avg| avg.reward_multiplier());
+                    .map_or(Decimal::default(), |avg| avg.reward_multiplier());
                 shares *= speedmultiplier;
                 (pubkey, shares)
             })
-            .filter(|(_pubkey, shares)| shares > &dec!(0.0))
+            .filter(|(_pubkey, shares)| shares > &Decimal::default())
```

Comment:
did clips suggest this change? I think the comparison against 0 is more clear. Or better yet (untested):

```suggestion
            .filter(|(_pubkey, shares)| shares > &Decimal::ZERO)
```

Reply:
no clippy suggestion; i agree the zero constant is a clearer version of the same

Reply:
fixed [0d6e939](https://github.com/helium/oracles/pull/191/commits/0d6e93943e8c0f7ad74905f4b2a8e50eae0ea4b8)